classes/SpotifyFeed.php

```diff
@@ -41,6 +41,23 @@ private function verifySong(string $song_id, string $user){
     }
     
     /**
+     * Accept a valid username and a specific song's id
+     * and save it into the user's liked songs list
+     *
+     * @param string $song_id The Spotify song id code
+     * @param string $user The current user's username
+     */
+    public function saveSong(string $song_id, string $user){
```

Comment:
Follow PSR-2 for code standards.  You should implement a code sniffer.  You can refer to some other repos to see how it is done, and copy the code sniffing settings.  In this particular case, `{` would be moved to the next line.